sklearn/covariance/tests/test_robust_covariance.py

```diff
@@ -40,6 +42,17 @@ def test_mcd():
     launch_mcd_on_dataset(500, 1, 100, 0.001, 0.001, 350)
 
 
+def test_fast_mcd_on_invalid_input():
+    X = np.arange(100)
+    assert_raise_message(ValueError, 'expects at least 2 samples', fast_mcd, X)
+
+
+def test_mcd_class_on_invalid_input():
+    X = np.arange(100)
+    mcd = MinCovDet()
+    assert_raise_message(ValueError, 'expects at least 2 samples', mcd.fit, X)
```

Comment:
Please check for the presence of the class name in the message: `'MinCovDet expects at least 2 samples'`.